activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,46 @@
+require 'active_support/time'
+
+module ActiveSupport  
+  class Claims
+    class InvalidClaims < StandardError; end
+    class ExpiredClaims < StandardError; end
+    
+    attr_reader :payload, :purpose, :expires
+
+    def initialize(options)
+      @payload = options.fetch(:value)
+      @purpose = self.class.pick_purpose(options)
+      @expires = self.class.pick_expiration(options)
+    end
+
+    class << self
+      def pick_purpose(options)
+        options.fetch(:for) { 'universal' }
+      end
+
+      def pick_expiration(options)
+        return options[:expires] if options.key?(:expires)
```

Comment:
I still think we should support both `expires_at` and `expires_in` like a signed Global ID does. `expires` would be considered an alias for `expires_at`.